src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/WrappedKeySecurityToken.cs

```diff
@@ -64,7 +96,23 @@ internal WrappedKeySecurityToken(string id, byte[] keyToWrap, string wrappingAlg
             this.wrappingAlgorithmDictionaryString = wrappingAlgorithmDictionaryString;
         }
 
-        public override string Id => this.id;
+        internal WrappedKeySecurityToken(
+          string id,
+          byte[] keyToWrap,
+          string wrappingAlgorithm,
+          SecurityToken wrappingToken,
+          SecurityKeyIdentifier wrappingTokenReference,
+          byte[] wrappedKey,
+          SecurityKey wrappingSecurityKey)
+          : this(id, keyToWrap, wrappingAlgorithm, (XmlDictionaryString)null, wrappingToken, wrappingTokenReference, wrappedKey, wrappingSecurityKey)
+        {
+        }
+
+
+        public override string Id
+        {
```

Comment:
You've replace an expression based property with an explicit property definition. The original one you are replacing is better.

Reply:
done